extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -20,6 +20,16 @@ public function __construct() {
 	}
 
 	/**
+	 * Check if a user is an admin
+	 *
+	 * @param string $group The user's group from ug_group
+	 * @return bool
+	 */
+	public function isAdmin( $group ) {
+		return strpos( $group, 'sysop' ) !== false;
```

Comment:
Why `strpos`? `ug_group` should be an exact group name so `$group === 'sysop'` should work.